gdal/ogr/ogrsf_frmts/flatgeobuf/ogrflatgeobuflayer.cpp

```diff
@@ -39,12 +39,17 @@ using namespace flatbuffers;
 using namespace FlatGeobuf;
 
 static OGRGeometry *CPLErrorInvalidPointer() {
-    CPLError(CE_Failure, CPLE_AppDefined, "Possible data corruption - unexpected nullptr");
+    CPLError(CE_Failure, CPLE_AppDefined, "Unexpected nullptr - possible data corruption");
```

Comment:
if this method return std::nullptr_t , then you should be able to save the explicit casts when using it

Reply:
Ah - did read about std::nullptr_t but got confused, I see now it works well for this case. Tthanks!